azure/iac/master-compliance-test.json

```diff
@@ -2674,6 +2674,130 @@
                             ]
                         }
                     ]
+                },
+                {
+                    "masterTestId": "TEST_sql_servers_encryption",
+                    "masterSnapshotId": [
+                        "ARM_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(sql_servers_encryption.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0111-ARM",
+                            "eval": "data.rule.serverKeyType",
+                            "message": "data.rule.serverKeyType_err",
+                            "remediationDescription": "Make sure SQLServer/encryptionProtector has ( properties.serverKeyType ='AzureKeyVault' )",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.